clippy_lints/src/needless_pass_by_ref_mut.rs

```diff
@@ -93,6 +93,16 @@ fn should_skip<'tcx>(
     is_from_proc_macro(cx, &input)
 }
 
+fn inherits_cfg(tcx: TyCtxt<'_>, def_id: LocalDefId) -> bool {
```

Comment:
This would be cleaner with `tcx.hir().parent_iter()`, Which is what [`is_in_cfg_test`](https://doc.rust-lang.org/nightly/nightly-rustc/src/clippy_utils/lib.rs.html#2430-2446) does. The lint already gets the `HirId` of the function so that can be turned into a local.

(PS: This should probably be a utils function similar to that one)

Reply:
Sounds good to me. Moving to the function to utils.